content/c++/encyclopedia/exceptions-in-cpp.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: "Exceptions in C++"
+Subjects:
+  - "Computer Science"
+  - "Game Development"
+Tags:
+  - "Catch"
+  - "Errors"
+  - "Try"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-c-plus-plus"
+  - "https://www.codecademy.com/learn/paths/computer-science"
+  - "https://github.com/Codecademy-Curriculum/codepedia-content/blob/main/content/language-agnostic/encyclopedia/what-is-an-error.md"
+---
+
+## Catching an Exception
+
+The function `getString` as defined below will throw an error if you try to access an index outside the allowable bounds.
+
+```codebyte/cpp
+std::string getString(std::string values[], int index) {
+    return values[index];
+}
+
+int main()
+{
+    std::string words[] = { "one", "two", "three" };
```

Comment:
Let's take out the extra space before the first element and after the last element.